bridge_common.spec

```diff
@@ -0,0 +1,58 @@
+Name: tendrl-bridge-common
+Version: 0.0.1
+Release: 1%{?dist}
+BuildArch: noarch
+Summary: Common Module for all Bridges
+Source0: %{name}-%{version}.tar.gz
+Group:   Applications/System
+License: LGPLv2+
+URL: https://github.com/Tendrl/bridge_common
+
+BuildRequires: systemd
+BuildRequires: python2-devel
+BuildRequires: pytest
+
+Requires: python-etcd
+Requires: python-dateutil
+Requires: python-hacking
+Requires: pytest
+Requires: python-pytest-cov
```

Comment:
1) Sure, I agree, I will move "python-hacking, pytest, python-pytest-cov to build requires".
2) yes i tried mock build for epel7 using the following command you have suggested and its succeeded.
mock -r epel-7-x86_64 rebuild /home/tim/rpmbuild/SRPMS/tendrl-gluster-bridge-0.0.1-1.fc22.src.rpm